[DOC-3744] SMP 0.8.3, 798xx patch release notes (#3348)
harness/developer-hub: release-notes/self-managed-enterprise-edition.md

title: Self-Managed Enterprise Edition release notes
sidebar_label: Self-Managed Enterprise Edition
tags: [NextGen, "self-managed-ee"]

> date: 2023-09-19T10:00

Author: ```suggestion
date: 2023-09-20T10:00
```

Patch releases for Harness Self-Managed Enterprise Edition include minor bug fixes and updates to address potential security vulnerabilities.

##### 0.8.3

This release includes the following Harness module and component versions.

| **Name** | **Version** |
| :-- | :--: |
| Helm Chart | [0.8.3](https://github.com/harness/helm-charts/releases/tag/harness-0.8.3) |
| Air Gap Bundle | [0.8.3](https://console.cloud.google.com/storage/browser/smp-airgap-bundles/harness-0.8.3) |

> | NG Manager | 79821 |

Reviewer: This should be ng-manager-signed:79823

> | CI Manager | 4903 |

Reviewer: This should be ci-manager-signed:4905

| Pipeline Service | 1.37.13 |
| Platform Service | 79601 |
| Access Control Service | 79400 |
| Change Data Capture | 79819 |
| Test Intelligence Service | release-177 |

> | NG UI | 0.353.10 |

Reviewer: This should be nextgenui-signed:0.353.17

| LE NG | 68004 |

**Alternative air gap bundle download method**

Some admins might not have Google account access to download air gap bundles. As an alternative, you can use `gsutil`. For `gsutil` installation instructions, go to [Install gsutil](https://cloud.google.com/storage/docs/gsutil_install) in the Google Cloud documentation. 

```
gsutil -m cp \
  "gs://smp-airgap-bundles/harness-0.8.3/ccm_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/cdng_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/ce_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/cet_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/ci_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/ff_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/platform_images.tgz" \
  "gs://smp-airgap-bundles/harness-0.8.3/sto_images.tgz" \
  .
```

#### New features and enhancements

> #### Self-Managed Enterprise Edition

Reviewer 2: these heading levels are the same. Is Self-Managed EE supposed to be subordinate to New Features and enhancements?

Author: Nice catch. Thanks, @michaelcretzman

> #### Self-Managed Enterprise Edition

Author: ```suggestion
##### Self-Managed Enterprise Edition
```

> - Updated CI Manager and the delegate to improve performance testing.(SMP-2022)

Reviewer 2: ```suggestion
- Updated CI Manager and the delegate to improve performance testing. (SMP-2022)
```

- Added the following rule to the `log-service` virtual service for performance testing. (SMP-2033)

   ```yaml
   - uri:
        prefix: /log-service//
   ```

> #### Cloud Cost Management

Author: ```suggestion
##### Cloud Cost Management
```

- Updated the Helm chart to support proxy configuration for AWS SDK clients. (CCM-14374)

- Updated the Helm chart to support AWS regions other than the default `us-east-1` region. (CCM-14415)
   
   ```

>    S3

Reviewer 3: We can remove this block of endpoint urls from here. 621 to 637

   s3.<region>.amazonaws.com
   <bucket>.s3.<region>.amazonaws.com

   STS
   sts.<region>.amazonaws.com

   EC2
   ec2.<region>.amazonaws.com

   ECS
   ecs.<region>.amazonaws.com

   cloudwatch
   monitoring.<region>.amazonaws.com
   ```

> #### Chaos Engineering

Author: added

> #### Chaos Engineering

Author: ```suggestion
##### Chaos Engineering
```

- Added support for Chaos Engineering version 1.19.x. (CHAOS-2544)

#### Continuous Delivery

> - Added the GitOps feature flag at the org level. (CDS-79545)

Reviewer 4: Added the functionality to support Gitops section at Org Level

> #### Continuous Delivery

Author: ```suggestion
##### Continuous Delivery
```

> #### Continuous Integration

Author: ```suggestion
##### Continuous Integration
```

> - Added support to set the `LOG_STREAMING _URL` from Harness Delegate by default for CI pipelines. If the delegate is outside of the cluster, the fallback log service URL is `LOG_SERVICE_ENDPOINT` from CI. (CI-9262, ZD-48974)

Reviewer 2: ```suggestion
- To support  CI pipelines, Harness added support to set the Harness Delegate environment variable `LOG_STREAMING _URL` by default. If the delegate is outside of the cluster, the fallback log service URL is `LOG_SERVICE_ENDPOINT` from CI. (CI-9262, ZD-48974)
```

> #### Custom Dashboards

Author: ```suggestion
##### Custom Dashboards
```

- Updated to an FIPS compliant algorithm for short term caching key generation. (CDB-1107)

#### Early access features

This release does not include any early access features.

#### Fixed issues

> #### Continuous Integration

Author: ```suggestion
##### Continuous Integration
```
